src/app/base/components/sidebar-pm/sidebar-pm.component.html

```diff
@@ -153,6 +153,23 @@
                             </a>
                         </div>
                     </div>
+                    <div class="flex items-center justify-center overflow-visible mt-12">
+                        <div data-tip="Join our community" class="tooltip tooltip-right relative z-50"
+                            data-tooltip-target="tooltip-right" data-tooltip-placement="right">
```

Comment:
we sometimes use data-tooltip-target and sometimes class tooltip-right. As far as I know, only the tooltip-right is needed to place it right. what is the other part for?

Reply:
I think the data-tooltip-target should be removed on all places, I removed the part and it nothing is changed since the tooltip-right does it work. If it okay I will remove it everywhere as part of the PR.

Reply:
lets go for it